static/includes/General/iXsystemsSupportContact.html.part

```diff
@@ -1,3 +1,5 @@
+Customers who purchase iXystems hardware or that want additional support must have a support contract to use iXystems Support Services. 
```

Comment:
Suggesting an additional sentence to remind the free users the forums are available for support.
```suggestion
Customers who purchase iXystems hardware or that want additional support must have a support contract to use iXystems Support Services. The [TrueNAS Community forums](https://www.truenas.com/community/) provides free support for users without an iXsystems Support contract.
```